### PR TITLE
Deletelast

### PR DESCRIPTION
**Pull request by aiwhoo**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -71,6 +71,14 @@ def clear_labels(self):
             self.delete(text_id)
         self.labels = []
 
+    #STEPHEN MOSS ADDITION
```
Comment (aiwhoo) [inline main.py]: No need to add this comment since it's stored in the github meta.

File: main.py [hunk 2]
```diff
@@ -71,6 +71,14 @@ def clear_labels(self):
...
+    def clear_last_label(self):
+        print(self.labels)
+        bb, class_name, bb_id, text_id = self.labels[-1]
```
Comment (aiwhoo) [inline main.py]: What if self.labels = []?

Comment (SteRMo95) [reply main.py]: fixed that in latest update

File: main.py (hunk 1 above)
Comment (SteRMo95) [reply main.py]: fair point, derp

**Pull request merged**